evaluators/anthropic/claude-arch/CHANGELOG.md

```
@@ -0,0 +1,11 @@
+# Changelog — claude-arch
+
+## [1.0.0] - 2026-04-16
+
+### Added
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)